packages/ts/react-auth/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This should be copied from e.g. react-grid instead

Reply:
Copied from `react-form`. I preferred not to alter the global config.